knot.x/knotx/conf/routes/operations.conf

```diff
@@ -42,4 +42,61 @@ routingOperations = ${routingOperations} [
       }
     ]
   }
+
+  {
+    operationId = scenario0
+    handlers = ${config.server.handlers.common.request} [
+      {
+        name = httpRepoConnectorHandler
+        config = {include required(classpath("routes/handlers/httpRepoConnectorHandler.conf"))}
+        config = {
+          clientDestination {
+            domain = ${mocks.domain}
+          }
+        }
+      },
+      {
+        name = fragmentsProviderHtmlSplitter
+      },
+      {
+        name = fragmentsHandler
+        config = {include required(classpath("routes/handlers/scenario0.conf"))}
+        config = {
+          actions {
+            user {
+              config {
+                endpointOptions {
+                  domain = ${mocks.domain}
```

Comment:
I understand that this solution is connected with HOCON limitations. We could not have the next "include" level with  substitutions.

Reply:
In that case, maybe we should keep it with dot notation?
`user.conifg. endpointOptions.domain = ${mocks.domain}`

Reply:
@Skejven It works with dot notation. Changed.